pkgs/desktops/deepin/deepin-terminal/default.nix

```diff
@@ -23,6 +24,7 @@ stdenv.mkDerivation rec {
     vala
     gettext
     gobject-introspection # For setup hook
```

Comment:
Vala has a setup hook now so you should try to remove this where you can.

Reply:
Done.